packages/search-adapter/src/empathy/mappers/request/params/empathy-request-related-tags-query.mapper.ts

```diff
@@ -8,9 +9,30 @@ import { RequestMapper } from '../../../empathy-adapter.types';
  * @public
  */
 @injectable()
-export class EmpathyRequestRelatedTagsQueryMapper implements RequestMapper<QueryableRequest, string> {
-
+export class EmpathyRequestRelatedTagsQueryMapper
+  implements RequestMapper<QueryableRequest, string>
+{
   map({ relatedTags = [] }: QueryableRequest, query: string): string {
-    return relatedTags.reduce((chain, rt) => `${chain} ${rt.tag}`, query).trim();
+    const [leftRts, rightRts] = this.splitRelatedTagsByQueryPosition(
+      relatedTags,
+      query
+    );
+    return `${leftRts} ${query} ${rightRts}`.trim();
+  }
+
+  splitRelatedTagsByQueryPosition(
+    relatedTags: RelatedTag[],
+    query: string
+  ): string[] {
+    return relatedTags
+      .reduce(
+        ([left, right], rt) => {
```

Comment:
Avoid using abbreviations as variable names `rt`, `rts` are not valid names following our naming conventions.